_includes/hub/footer.html

```diff
@@ -42,7 +42,7 @@
 			</div>
 		</div>
 		<div class="siteFooter__bottom">
-			<p class="footer-copyright"><span class="terms-text"><a href="https://www.sfu.ca/contact/terms-conditions.html" target="_blank">TERMS & CONDITIONS</a> &copy; SIMON FRASER UNIVERSITY.</span><span class="copyright-text"><img src="/img/creativecommons.svg" alt="Creative Commons logo" /><img src="/img/attribution.svg" alt="Attribution icon" /><a href="https://creativecommons.org/licenses/by/2.0/" target="_blank">CC-BY 2.0</a></span><span class="footer-copyright-separator"> | </span><span class="ab-link">Website by <a target="_blank" href="https://www.affinitybridge.com">Affinity Bridge</a></span></p>
+			<p class="footer-copyright"><span class="terms-text"><a href="https://www.sfu.ca/contact/terms-conditions.html" target="_blank">TERMS & CONDITIONS</a> &copy; SIMON FRASER UNIVERSITY.</span><span class="copyright-text"><img src="/img/creativecommons.svg" alt="Creative Commons logo" /><img src="/img/attribution.svg" alt="Attribution icon" /><a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">CC BY 4.0<span class="-screenReader"> - Attribution 4.0 International </span></a></span><span class="footer-copyright-separator"></span></p>
```

Comment:
Hi @israelcefrin, is `<span class="footer-copyright-separator"></span>` still needed here? It was used to apply styling to the `|` character previously.